app/components/LeftBubble.tsx

```diff
@@ -21,13 +21,16 @@ export const LeftBubble = ({ message }: { message: Message }) => {
               </div>
             </div>
           </div>
-          <div className="h-6 w-6 shrink-0 self-center">
+          <div className="hidden md:block h-6 w-6 shrink-0 md:self-center">
             <MessageAudio message={message} />
           </div>
         </div>
       </div>
-      <div className="col-start-1 col-end-13 px-3 pb-3">
-        <MessageMeta className="ml-14" message={message} />
+      <div className="col-start-1 col-end-13 md:px-3 pb-3 flex gap-2">
+        <div className="md:hidden h-6 w-6 shrink-0 md:self-center">
+          <MessageAudio message={message} />
```

Comment:
This needs fixing, its creating bug, where is user plays the voice over the desktop and resize to mobile then since its a different component with different node, the voice state is not maintained, i.e., if user resize the window to mobile it the icon will be of pause but voice will still play

Reply:
Need to think of some other solution.